LeaderboardBackend/openapi.json

```diff
@@ -2534,7 +2548,8 @@
           "Time",
           "Score"
         ],
-        "type": "string"
+        "type": "string",
+        "description": "<p>Members:</p><ul></ul>"
```

Comment:
This doesn't looks right.

Reply:
I didn't consider our existing enums not having documentation. Should I just drop the schema filter? I'm happy to add documentation to `RunType` and extend the filter to handle empty cases later on, though.

Reply:
Dropping the filter